babybuddy/settings/base.py

```diff
@@ -351,9 +352,13 @@
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
 # Baby Buddy configuration
-# See README.md#configuration for details about these settings.
+# See https://docs.baby-buddy.net/ for details about these settings.
 
 BABY_BUDDY = {
     "ALLOW_UPLOADS": bool(strtobool(os.environ.get("ALLOW_UPLOADS") or "True")),
     "READ_ONLY_GROUP_NAME": "read_only",
 }
+
+# Home assistant specific configuration
+
+HOME_ASSISTANT_USE_X_INGRESS_PATH = False
```

Comment:
We'll want to expose this via an env variable. See e.g. `SESSION_COOKIE_SECURE` for an example.

Reply:
Little clarification: I now use a custom `homeassistant.py` config with these contents:

~~~~~
from .base import *

HOME_ASSISTANT_USE_X_INGRESS_PATH = True
~~~~~

to do the same thing. You would prefer env-vars for this? I could also add the custom settings-file alternatively? Or do you prefer envvars generally?

Reply:
I mostly rely on env vars to ensure Docker users are supported without having to fiddle with file management and storage stuff. If HA is not via Docker (is it, I thought it was?) I suppose it's OK to keep this without an env var.

Reply:
Nah, it is docker based. So an env-var party it is!